meta-resin-common/recipes-support/balena-custom-udev-rules/balena-custom-udev-rules/balena-custom-udev-rules

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+. /usr/sbin/resin-vars
+
+keys=`jq -r 'select(.settings != null) | select(.settings.udevrules != null) | .settings.udevrules | keys[] ' $CONFIG_PATH`
+RELOAD_RULES=0
+lscmd="ls "
+
+if [ ! -z "$keys" ]; then
+	echo "Found custom udev rules in $CONFIG_PATH"
+	while read -r key; do
+		echo "Found custom udev rule: $key"
+		lscmd="$lscmd -I \"$key.rules\""
+		filter="'.settings.udevrules.\"$key\" []'"
+		cmd="jq -r $filter $CONFIG_PATH"
+		rule=$(eval "$cmd")
+		if [[ -f "/etc/udev/rules.d/$key.rules" ]] && [[ $(< "/etc/udev/rules.d/$key.rules") = "$rule" ]]; then
```

Comment:
[[ is not defined by POSIX - lets avoid bashisms.

Reply:
Changed to using cat and `[ ]`